### ISSUE TITLE
Instant save, without asking for editing

### ISSUE THREAD

**Issue by kaldown** [flameshot-org/flameshot#1576]
https://github.com/flameshot-org/flameshot/issues/1559#issuecomment-825200419

It would be cool to select an area and just save it without questioning to edit in a `gui` mode

**Comment by mmahmoudian** [comment]
I think this will be automatically resolved when #1531 is addressed as it would become more intuitive.
You can do `flameshot gui -p /path/to/save` and after #1531 there will be a "capture button" (name is not confirmed yet) instead of having copy and save buttons. This way users can just assign that key to Enter or something.

What do you think? Does this address this issue?

**Comment by kaldown** [comment]
Thanks for describing.
But, I think it is not.
Since with this button I still have to click something.

Mind that when you selecting regions, on 90% of the cases, you already know what the result should be look like.
There is no point to hit any button after visual select ends.
Otherwise, just re-draw your rectangle.

That's the idea.

**Comment by mmahmoudian** [comment]
Actually for my use-case, it is typically not easy to draw the rectangle with enough precision as I should mind all 4 sides of it. Therefore in my workflow, I always have to fine-tune the edges to make sure it is capturing everything I need. Also I personally don't mind pressing a keystroke (e.g Enter) to signify that I'm done and I want the screenshot to be taken.

Regardless, I think if a CLI argument is added to bypass all annotation parts and etc, (combined with `-c`, `-p`, or `-r`) that would be useful for automation and integration into workflows. This can be extra useful if the coordinates of the selection area is defined in CLI (#479 and #558).

I'll label this as a feature request.

**Comment by Craluminum2413** [comment]
Would be nice to have it as second option, if you need to make fast screenshot of the whole screen without editing, like in the lightshot. One for editing, one for make immediately screenshot and save in the selected earlier directory, and one for make screenshot for copy

**Comment by borgmanJeremy** [comment]
I don't think we will add this since you can already just hit a hotkey to save or copy the image.
